validator/src/server/cli.rs

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright 2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ------------------------------------------------------------------------------
+ */
+
+use clap::{App, Arg, ArgMatches};
+use cpython::{PyDict, Python};
+
+const DISTRIBUTION_NAME: &str = "sawtooth-validator";
+const VERSION: &str = env!("CARGO_PKG_VERSION");
+
+pub fn get_cli_args(py: &Python) -> PyDict {
```

Comment:
This should take `matches` as an argument and return PyResult<PyDict> instead of unwrapping. The caller can then do `.map_err(|err| err.print(*py))` to get a nice python stack trace if there was a problem. Doing an unwrap on a PyResult causes very opaque errors.